app/persistence/transaction/transaction.go

```diff
@@ -44,6 +44,7 @@ type Transaction struct {
 	SubmissionTxId string
 	Status         string
 	EthHash        string
+	Asset          string
```

Comment:
I am wondering whether it will be better to populate `asset` with `HBAR` (for Hbars) and `0.0.15` for tokens OR populate the `erc20-contract-address`.

@georgiyazovaliiski @failfmi wdut?

Reply:
If you would like to have information for both the asset on Hedera(either `HBAR` or `TokenID`) and the corresponding `ERC20`, we can introduce two properties.

Reply:
having both might be redundant... I would go for `HBAR` / `0.0.15` compared to `ERC20-contract-address`. 
+ we will have `erc20-address` as column in the `messages` records (since it will be signed)